main.rb

```diff
@@ -6,6 +6,13 @@
 
 java_import "org.yaml.snakeyaml.Yaml"
 
+# solrj stuff
+java_import "org.apache.solr.client.solrj.SolrServerException"
+java_import "org.apache.solr.client.solrj.impl.HttpSolrServer"
```

Comment:
Looks like we will probably need SolrCore as well (https://github.com/apache/zest-java/blob/f21cc656a72b6fb5fdca112b05133d7e63455a3d/extensions/indexing-solr/src/main/java/org/apache/zest/index/solr/EmbeddedSolrService.java#L28) (https://github.com/apache/zest-java/blob/f21cc656a72b6fb5fdca112b05133d7e63455a3d/extensions/indexing-solr/src/main/java/org/apache/zest/index/solr/EmbeddedSolrService.java#L83) and SolrServer